Modules/Typescript/Typescript-Features/homework.js

```diff
@@ -0,0 +1,113 @@
+console.log(`
+  Task
+    1. Create class, which extends from an AbstractClass
+    2. Apply interfaces to class, it's method, property
+    3. Use generics there
+    4. Analyze transpiled code in ES5 and explain it
+`);
+var LimbEnum;
```

Comment:
this is `enum` -> variable initialised as object with parameters/keys with specific values (key -> `WING`, value -> `Wing`)